### PR TITLE
Lanelet handler

### PR DESCRIPTION
**Pull request by AnonymDavid**
A package that loads a lanelet2 map from a file and publishes it to a specified topic as binary data.

### CODE REVIEW

**Comment by AnonymDavid** [comment]
Just realized I need to convert the output topics of the driver to match the predefined ones (sensor abstraction inputs).

File: crp_VIL/lanelet_handler/launch/laneletFileLoader.launch.py
Comment (mesmatyi) [inline crp_VIL/lanelet_handler/launch/laneletFileLoader.launch.py:1]: would be nice to have an option to set the frame_id, by default it's set to **map** but in some cases could be **map_zala_0**

**Review by mesmatyi — CHANGES_REQUESTED** [review]

**Comment by mesmatyi** [comment]
I feel also the Lanelet2 visualizer package is needed, https://github.com/autowarefoundation/autoware.universe/tree/main/map/map_loader#lanelet2_map_visualization

File: crp_VIL/lanelet_handler/launch/laneletFileLoader.launch.py
Comment (AnonymDavid) [reply crp_VIL/lanelet_handler/launch/laneletFileLoader.launch.py:1]: Done! ([commit](https://github.com/jkk-research/CooperativeResearchPlatform/pull/14/commits/58e0b5847c6ffd836e07879596cd2fb76155ae93))

**Comment by AnonymDavid** [comment]
> I feel also the Lanelet2 visualizer package is need to be caried over, https://github.com/autowarefoundation/autoware.universe/tree/main/map/map_loader#lanelet2_map_visualization

I added the autoware visualizer to the launcher.

**Review by mesmatyi — APPROVED** [review]

**Review by gfigneczi1 — APPROVED** [review]
from my side it's ok, can be merged

**Pull request merged**